model_building/model_building.py

```diff
@@ -134,9 +134,10 @@ def process(self, campaign_configuration, regression_inputs, processes_number):
                         self._logger.debug("Wrapper: "+str(exp.get_x_columns()))
                         if exp.is_wrapper():
                             self._logger.debug("Wrapped: "+str(exp._wrapped_experiment_configuration.get_x_columns()))
-                    except KeyboardInterrupt as ki:
+                    except KeyError as ki:
```

Comment:
Why does this fix the `KeyboardInterrupt` problem? Before the change, the code would raise any `KeyboardInterrupt` and ignore other exceptions (line 141). A `KeyError` is raised e.g. when a key is not found in a dictionary, `DataFrame`, etc. So this has nothing to do with the `KeyboardInterrupt`

Reply:
The exception raised is always a KeyError, likely due to mlxtend code in the training of an sfs model that filters the exception: is a submodule a better solution?